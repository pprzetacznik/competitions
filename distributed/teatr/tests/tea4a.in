90000000 0 -1 -1 1000000 17318417819340455879
