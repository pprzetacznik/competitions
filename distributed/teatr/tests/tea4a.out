2024999627793587
